scripts/tools.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+brew install ct
+brew install yq
+brew install norwoodj/tap/helm-docs
```

Comment:
Maybe we could use this 🤔 

`brew install --cask \
cask1 \
cask2 \
cask3 \
...
`

Reply:
:shrug: 
Just did oneliner. And you got other bug I missed :D name of the pacakge